app/templates/index.hbs

```diff
@@ -0,0 +1,4 @@
+<div id="welcome">
+<h1 id = "tajtl"> {{t 'welcome.to.coolwriter.editor'}} </h1>
```

Comment:
Don't add extra space after HTML tag and before attributes in HTML tag. Keep everything slim if you can.